keyboards/viktus/osav2_numpad/info.json

```diff
@@ -0,0 +1,242 @@
+{
+    "manufacturer": "Viktus Design LLC",
+    "keyboard_name": "OSAv2 Numpad",
+    "maintainer": "BlindAssassin111",
+    "url": "https://viktus.design",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x5644",
+        "pid": "0x4F4E"
+    },
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["F5", "F0", "D3", "D5"],
+        "rows": ["F1", "F4", "D4", "D7", "D6"]
+    },
+    "rgblight": {
+        "led_count": 9,
+        "pin": "C7",
+        "max_brightness": 191,
+        "animations": {
+            "breathing": true,
+            "rainbow_swirl": true,
+            "twinkle": true
+        }
+    },
+    "layouts": {
+        "LAYOUT_all_split": {
+            "layout": [
+                { "label": "K00", "matrix": [0, 0], "w": 1, "x": 0, "y": 0 },
```

Comment:
```suggestion
                { "label": "K00", "matrix": [0, 0], "x": 0, "y": 0 },
```
etc

Reply:
Would this apply to all boards with `"w": 1`? Unsure as it was fine in all of the other PRs I have done in the past month.

Reply:
The default width and height of a key is 1u, so they can be omitted unless they are != 1.